src/vs/workbench/contrib/extensions/browser/extensions.contribution.ts

```diff
@@ -261,7 +261,12 @@ Registry.as<IConfigurationRegistry>(ConfigurationExtensions.Configuration)
 				type: 'boolean',
 				description: localize('extensionsInQuickAccess', "When enabled, extensions can be searched for via Quick Access and report issues from there."),
 				default: true
-			}
+			},
+			'extensions.experimental.supportWorkspaceExtensions': {
```

Comment:
Do you not need to tag this setting as 'experimental' in-case you're planning on enabling this via experiments?

Reply:
Good point. But as of now, we are not planning to enable experiments on this setting.